src/main/java/athena/task/Task.java

```diff
@@ -61,6 +63,9 @@ public Task(String name, String startTime, String duration, String deadline,
         this.notes = notes;
         this.number = number;
         this.isFlexible = isFlexible;
+        if (recurrence.equals("TODAY")) {
+            recurrence = DateUtils.formatDate(LocalDate.now());
+        }
```

Comment:
Remember to remove the `case "TODAY"` when we check for `switch (recurrence)` in `setRecurrence()` in the `Time` class